py/client/setup.py

```diff
@@ -37,7 +37,7 @@ def _compute_version():
     description='The Deephaven Python Client',
     long_description=_get_readme(),
     long_description_content_type="text/markdown",
-    packages=find_packages(exclude=("tests",)),
+    packages=find_namespace_packages(exclude=("tests","examples","docs","build"),),
```

Comment:
Hmm... I see docs here, but `docs/source/conf.py ` is _not_ being excluded, I'm guessing b/c it has a `.py` extension.

Reply:
Apparently the exclude isn't recursive, will investigate.

Reply:
The directory structure doesn't contain `__init__.py` files, so I wonder if it is not treating it as a package and therefore not excluding.  Just a wild guess.

Reply:
Right, we only want it to be a namespace, not a module - if we required that there be a `__init__.py` file, that could conflict with other wheels from contributing to it (at least without stomping on each other).